CONTRIBUTING.md

```diff
@@ -50,34 +50,36 @@ export PATH="$PATH:$(go env GOPATH)/bin"
 Once the `*.pb.go` files are generated, you need to replace the content of the `api/collector/arrow/v1` directory by the
 generated files present in the `./proto/api/collector/arrow/v1` directory.
 
+### Releasing this repository
+
+See the instructions in [RELEASING.md][./RELEASING.md].
+
 ### Local development issues
 
-The network of dependencies involved in building OpenTelemetry
-Collector images has at times pushed the `go mod` toolchain to its
-limits.  While we would like to recommend the `go work` tool for local
-development, there are currently unresolvable dependency problems that
-happen as a result of this.
-
-The traditional solution to multi-module repositories before `go
-work` was introduced is the Go module `replace` statement, which
-allows mapping inter-repository dependencies to local directory paths,
-allowing you to build and test an OpenTelemetry collector with
-locally-modified sources.
-
-While the use of replace statements works to enable local development,
-it prevents running code directly from the repository, which raises a
-barrier to entry.  To work around this problem, the checked-in
-contents of `./collector/cmd/otelarrowcol/go.mod` must not contain
-`replace` statements.  To build an `otelarrowcol` from locally
-modified sources requires uncommenting the `replaces` directive in
-`./collector/cmd/otelarrowcol/build-config.yaml` and re-running `make
-genotelarrowcol otelarrowcol`.
+This repository contains a top-level `go.work` file, as an experiment.
```

Comment:
Is it still an experiment?

Reply:
Thanks. I think not, at this point. If we have troubles releasing v0.7 it'll be addressed in the next PR.